includes/class-wc-payments-account.php

```diff
@@ -1129,12 +1130,27 @@ private function init_stripe_onboarding( $wcpay_connect_from ) {
 			exit;
```

Comment:
This code will only run if the merchant has already onboarded but lost connection to jetpack. That's why the transient is only added after it